lib/wowrb.rb

```diff
@@ -15,16 +15,11 @@ def self.auction_url(realm_name)
     call_api(remote_url)
   end
 
-  def self.boss(boss_id)
+  def self.boss(boss_id = nil)
```

Comment:
Reflect this feature in readme to make clear which results will return method when boss id is passed and where it isn't.